pkg/tests/mappings/all_test.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for generated content manifest."""
+
+# TODO(aiuto): Generate this file.
+
+import unittest
+
+import manifest_test_lib
```

Comment:
Unnecessary whitespace?

Reply:
Our style is 
import .... standard python packages ...
\<blank line\>
import local pacakges